_docs/ios/1.158.0.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: 1.158.0
+permalink: /docs/ios/1.157.0/
+parent: iOS
+nav_order: -158
+release_date: 25.10.2022
+---
+
+## Devices can be set as "Favorite". Favorite devices will be displayed on top of the list in "My devices" and "Lock details" views.
+Each device can set us favorite in device settings. By default, when at least one device is marked as favorite, this device will present in favorite section in my devices page. This new section will contains all favorites devices sorted by name and device type. Also, in lock details page, favorites locks will be presented before other locks.\
```

Comment:
`us` > `as`

`will be presented in ...`

`will contain all favorited`

`by device type`

`favorited locks`